maps/aurora/aurora-4_mainlevel.dmm

```diff
@@ -7169,22 +7169,6 @@
 	dir = 1
 	},
 /obj/effect/floor_decal/industrial/outline/yellow,
-/obj/machinery/door/window/southleft{
-	dir = 4;
-	icon_state = "left";
-	name = "Atmospheric Hardsuits";
-	req_access = list(24)
-	},
-/obj/structure/table/rack,
-/obj/item/clothing/suit/space/void/atmos,
-/obj/item/clothing/head/helmet/space/void/atmos,
```

Comment:
What are these changes?

Reply:
Fixing a mapping bug. The issue #4829.

Reply:
They're fixes to an atmos duplicate items mappping error, per #4829

Reply:
Very atomic.